source/_components/getting-started.blade.php

```diff
@@ -1,10 +1,10 @@
 <section class="bg-gradient-purple py-16">
-    <div class="container-content flex flex-col items-center pt-32">
-        <h2 class="title-2 text-white mb-2">Getting started is easy</h2>
+    <div class="container-content flex flex-col items-center pt-32 pb-8">
+        <h2 class="text-white mb-2">Getting started is easy</h2>
 
         <p class="text-pink-lighter mb-8">Just make new project directory and install Jigsaw using Composer</p>
 
-        <code class="w-screen md:w-auto bg-opaque-purple py-3 sm:px-16 md:rounded">
+        <code class="w-screen md:w-auto py-3 sm:px-16 md:rounded-lg" style="background: rgba(45, 15, 52, 0.5);">
```

Comment:
can we use a color class here instead? `bg-purple-darker` (even if you have to adjust it in tailwind.js)

Reply:
Incidentally, `purple-lighter` is darker than `purple-light`

Reply:
I went with `purple-lighter here`